pandas/tests/frame/indexing/test_setitem.py

```diff
@@ -1235,3 +1235,21 @@ def test_setitem_not_operating_inplace(self, value, set_value, indexer):
         view = df[:]
         df[indexer] = set_value
         tm.assert_frame_equal(view, expected)
+
+    @td.skip_array_manager_invalid_test
```

Comment:
Yes, I think it indeed makes sense to skip this test instead of xfail, since the way is written is not intended to ever work for ArrayManager. But so we could indeed write a separate test that checks for the the ArrayManager case (but I would say it's maybe not a super high priority, I can only take a look in the weekend).

Reply:
> but I would say it's maybe not a super high priority

no problem. not a blocker.